althea_types/src/interop.rs

```diff
@@ -344,6 +364,14 @@ pub struct ExitList {
     pub wg_exit_listen_port: u16,
 }
 
+/// Struct returned when hitting exit_list_V2 endpoint
+#[derive(Default, Debug, Serialize, Deserialize, PartialEq, Eq, Hash, Clone)]
+pub struct ExitListV2 {
+    pub exit_list: Vec<ExitIdentity>,
+    // All exits in a cluster listen on same port
+    pub wg_exit_listen_port: u16,
```

Comment:
maybe make this per exit identity? Just to avoid forcing this assumption.

Reply:
added it to exitidentity